spring-integration-core/src/main/java/org/springframework/integration/support/management/graph/IntegrationGraphServer.java

```diff
@@ -291,13 +320,23 @@ private MessageHandlerNode polledHandlerNode(String name, PollingConsumer consum
 					? consumer.getDefaultErrorChannel().toString() : null;
 			MessageHandler handler = consumer.getHandler();
 			return handler instanceof CompositeMessageHandler
-					? compositeHandler(name, consumer, (CompositeMessageHandler) handler, outputChannelName, errorChannel,
```

Comment:
Let's rework this "ternary fractal" into `if...else`!
A bit complicated to review and IDEA is crazy refactoring them :smile: